types/shared/Responses.ts

```diff
@@ -6,13 +6,17 @@ import {URL} from './utils';
  */
 export type Responses = QuestionAndResponses[];
 
+export type QuestionTags = 'Sensitive data';
```

Comment:
It will mean a small mapping step in PlanX, but I think having these tags cased consistently will match the rest of the schema better, e.g. `"sensitiveData"`.

Reply:
Thanks for catching this ! Have actually decided that, similar to how we encode `Flags` in the schema, we can just introduce tags as a generic `Array<string>` type here - which will give us more flexibility / less new-release dependencies in future. We can include a couple planx-core tests to ensure this isn't actually _populated_ with stray values.